index.js

```diff
@@ -0,0 +1 @@
+const nombreUsuario = prompt("Ingrese su nombre")
```

Comment:
En este caso me parece que sería mejor que no uses interpolación de strings

```suggestion
alert("Hola " + nombreUsuario);
```